package.json

```
@@ -5,11 +5,16 @@
   "description": "Проект «Большое путешествие» от HTML Academy",
   "scripts": {
     "eslint": "eslint src/",
-    "test": "npm run eslint"
+    "test": "npm run eslint",
+    "build": "webpack --mode development",
```

build - это обычно продакшн сборка. Которая прям улетает конечным людям. 
```suggestion
    "build": "webpack --mode production",
    "develop": "webpack --mode development",
```

---

Но академия пока просит так, значит будем делать так. Почитать можно вот тут https://webpack.js.org/configuration/mode/

> Когда наш проект будет готов, мы перенастроим этот скрипт на сборку для публикации.